### ISSUE TITLE
Mylar - Should Update from Mylar GUI work?

### ISSUE THREAD

**Issue by ForumFerret** [SynoCommunity/spksrc#2491]
_For new Package Requests, see the guidelines_  
### Setup

Mylar
20160119-4

Syno DS716+
Intel Celeron
DSM 5.2-5967 Update 2
### Expected behavior

When viewing mylar GUI showing an update is available, clicking Update should update mylar to most recent commit by mylar developer
### Actual behavior

Update process appears to kick off, mylar GUI shows "Mylar is restarting", GUI re-appears at landing page with the same Update Available message displayed, and shows Mylar is running at the same version of code as before update was initiated. 
### Steps to reproduce

Install Mylar package
View Mylar GUI
Click Update 
#### Package log

_Check Package Center or `/usr/local/{package}/var/`_

```
13-Oct-2016 10:37:35 - INFO    :: CP Server Thread-6 : Retrieving latest version information from github
13-Oct-2016 10:37:36 - INFO    :: CP Server Thread-6 : New version is available. You are 4 commits behind
13-Oct-2016 10:37:56 - INFO    :: MAIN : Received signal: update
13-Oct-2016 10:37:56 - INFO    :: MAIN : Aborting all threads
13-Oct-2016 10:37:56 - INFO    :: MAIN : Waiting for the DB UPDATE thread to exit
13-Oct-2016 10:37:57 - INFO    :: MAIN : Waiting for the SEARCH thread to exit
13-Oct-2016 10:38:07 - INFO    :: MAIN : Waiting for the RSS CHECK thread to exit
13-Oct-2016 10:38:08 - INFO    :: MAIN : Waiting for the WEEKLY CHECK thread to exit
13-Oct-2016 10:38:09 - INFO    :: MAIN : Waiting for the VERSION CHECK thread to exit
13-Oct-2016 10:38:09 - INFO    :: MAIN : Waiting for the FOLDER MONITOR thread to exit
13-Oct-2016 10:38:09 - INFO    :: MAIN : Mylar is updating...
13-Oct-2016 10:38:10 - ERROR   :: MAIN : Git returned bad info. Are you sure this is a git installation?
13-Oct-2016 10:38:10 - ERROR   :: MAIN : Couldn't download latest version
13-Oct-2016 10:38:10 - WARNING :: MAIN : Mylar failed to update: 'NoneType' object has no attribute 'split'. Restarting.
13-Oct-2016 10:38:10 - INFO    :: MAIN : Removing pidfile /usr/local/mylar/var/mylar.pid
13-Oct-2016 10:38:10 - INFO    :: MAIN : Mylar is restarting...
```
#### Comments

should the update feature in Mylar be used, or should I wait for a new Syno package to be available and ignore updates between SynoCommunity releases?

**Comment by kianrafiee** [comment]
Have you resolved this issue? I am thinking about using mylar and currently cannot use synocommunity from package center on my new ds218play so I have to manually install packages.

**Comment by Safihre** [comment]
New test-package available in #3138. It will auto-update after the final release.
It works on all archs and auto-update also works.
